src/linear_algebra/basic.lean

```diff
@@ -1946,7 +1946,7 @@ protected def uncurry :
   (V → V₂ → R) ≃ₗ[R] (V × V₂ → R) :=
 { map_add'  := λ _ _, by { ext ⟨⟩, refl },
   map_smul' := λ _ _, by { ext ⟨⟩, refl },
-  .. equiv.arrow_arrow_equiv_prod_arrow _ _ _}
+  .. (equiv.curry _ _ _).symm }
 
 @[simp] lemma coe_uncurry : ⇑(linear_equiv.uncurry R V V₂) = uncurry := rfl
```

Comment:
In case my previous comment was unclear - I'm tempted to replace `linear_equiv.uncurry` with `linear_equiv.curry`, so that this line becomes:
```suggestion
@[simp] lemma coe_curry : ⇑(linear_equiv.curry R V V₂) = curry := rfl
```
and `equiv.curry` is consistent with `linear_equiv.curry`.

Reply:
I see. Let me try to refactor.

Reply:
Done